### PR TITLE
Hotfix : Fix regression to enable database metrics in telegraf only when needed

### PR DESCRIPTION
**Pull request by pappachino**
With micrometer metrics enabled by default for all apps (#481), database metrics related telegraf input plugin was enabled. This introduced a regression, as now for all apps we started running postgres queries. This change fixes the regression.

Going forward,
- For freeapps, we never include any database metrics via telegraf.
- For paidapps, any customers who have set DD_API_KEY or APPMETRICS_TARGET would get the database metrics via telegraf.

### CODE REVIEW

File: buildpack/telemetry/telegraf.py [hunk 1]
```diff
@@ -59,7 +59,18 @@ def _get_appmetrics_target():
...
+    if util.is_free_app():
+        # For free apps we are not interested in database metrics
+        return False
+
+    value = "false"
+    if _get_appmetrics_target() is not None or datadog.is_enabled():
+        # For customers who have Datadog or APPMETRICS_TARGET enabled,
+        # we always include the database metrics. They can opt out
+        # using the APPMETRICS_INCLUDE_DB flag
+        value = "true"
```
Comment (udh-mendix) [inline buildpack/telemetry/telegraf.py:71]: I think its better if the APPMETRICS_INCLUDE_DB check is inside this if block, so at the end you can simply return strtobool(value)

**Comment by udh-mendix** [comment]
The MR is not only about free apps, it fixes the regression which enabled database metrics in telegraf for ALL apps. Would be nice if its made clear in the MR description and title, thanks!

File: buildpack/telemetry/telegraf.py (hunk 1 above)
Comment (pappachino) [reply buildpack/telemetry/telegraf.py]: I've included the env var check inside the block now.

**Review by udh-mendix — COMMENTED** [review]
LGTM

**Review by udh-mendix — COMMENTED** [review]
LGTM

**Review by ajinkya-takle — DISMISSED** [review]
LGTM!

**Review by svanderburg — APPROVED** [review]

**Pull request merged**